### ISSUE TITLE
JWT Sauce key doesn't work with karma-sauce-launcher 4.1.5

### ISSUE THREAD

**Issue by choumx** [ampproject/amphtml#28821]
Creation of the Sauce connect tunnel works fine but face-plants when `karma-sauce-launcher` asks `webdriverio` to connect with user/key:

```
10 06 2020 19:06:55.024:ERROR [SaucelabsLauncher]: Error: A "user" or "key" was provided but could not be connected to a known cloud service (Sauce Labs, Browerstack or Testingbot). Please check if given user and key properties are correct!
    at detectBackend (/Users/willchou/amphtml/node_modules/@wdio/config/build/utils.js:94:11)
    at Object.remote (/Users/willchou/amphtml/node_modules/webdriverio/build/index.js:54:58)
    at runNextTicks (internal/process/task_queues.js:62:5)
    at listOnTimeout (internal/timers.js:518:9)
    at processTimers (internal/timers.js:492:7)
```

I verified that this works with my personal Sauce credentials but doesn't work with `user: amphtml` and a token from https://amphtml-sauce-token-dealer.appspot.com/getJwtToken. Reverting to `karma-sauce-launcher` 2.0.2 fixes it.

Moving forward, we probably can just hardcode our free-tier Sauce key for PR builds.

**Comment by rsimha** [comment]
This issue may have been fixed as of last week by #28838. The code that fetches a new JWT key was [replaced](https://github.com/ampproject/amphtml/pull/28838/files#diff-b7a8f991c70e3a8575b7404488fbc9f4L124) by a simple check of existing tokens set on Travis as environment vars.

**Comment by choumx** [comment]
Yup, and upgrade in #28906 works fine.
